LayoutTests/fast/multicol/column-break-serialization.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<script src="../../resources/testharness.js"></script>
+<script src="../../resources/testharnessreport.js"></script>
+<script>
+function* getLonghandValues() {
+    yield ["break-after", ["auto", "avoid", "avoid-column", "avoid-page", "column", "left", "page", "recto", "right", "verso"]];
+    yield ["break-before", ["auto", "avoid", "avoid-column", "avoid-page", "column", "left", "page", "recto", "right", "verso"]];
+    yield ["break-inside", ["auto", "avoid", "avoid-column", "avoid-page"]];
+}
+
+function* getShorthandValues() {
+    yield ["-webkit-column-break-after", ["always", "auto", "avoid"]];
+    yield ["-webkit-column-break-before", ["always", "auto", "avoid"]];
+    yield ["-webkit-column-break-inside", ["auto", "avoid"]];
+    yield ["page-break-after", ["always", "auto", "avoid", "left", "right"]];
+    yield ["page-break-before", ["always", "auto", "avoid", "left", "right"]];
+    yield ["page-break-inside", ["auto", "avoid"]];
+}
+
+function* getShorthandsFor(longhand) {
+    yield "-webkit-column-" + longhand;
+    yield "page-" + longhand;
+}
```

Comment:
Using generators here doesn't seem needed. You could just define variables right before they're used and iterate through them (either nested arrays, Object.entries(), etc.).

Reply:
Switched to maps.